packages/common/src/core/schemas/hubproject.ts

```diff
@@ -174,6 +177,17 @@ export const HubProjectEditUiSchema: IUiSchema = {
             },
           },
         },
+        {
+          labelKey: "{{i18nScope}}.featuredImage.label",
+          scope: "/properties/featuredImage",
+          type: "Control",
+          options: {
+            control: "hub-field-input-image-picker",
+            sizeDescription: "{{i18nScope}}.featuredImage.sizeDescription",
```

Comment:
Maybe you already know this, but I don't think think this will work currently. You'll need to add logic to the `image-picker` field to translate the `uiSchema.options.sizeDescription`. 

I'm also wondering if we should pass it in as an option similar to `helperText` where we do: 
```ts
{
  ...,
  options: {
    sizeDescription: {
      label?: "...", // un-translated label
      labelKey?: "..." // translated label key
    }
  }
}
```